spec/takeaway_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:dishes){ {chicken: 5, pizza: 3, ice_cream: 2} }
+
+  it 'has an empty order' do
+    expect(subject.order).to be_empty
+  end
+
+  it 'respond to menu' do
+    expect(subject).to respond_to(:menu)
+  end
+
+  it 'stores a list of dishes' do
+    expect(subject.menu).to eq dishes
```

Comment:
This test is tying you to a specific implementation.
You could make your code more reusable by not hard-coding the menu: https://github.com/nboyse/takeaway-challenge/blob/master/docs/review.md#open-closed-principle